src/viur/core/email.py

```diff
@@ -429,7 +429,7 @@ def check_sib_quota() -> None:
         entity["credits"] = credits
         entity["email"] = data["email"]
 
-        thresholds = sorted(conf.email.sendinblue_thresholds, reverse=True)
+        thresholds = sorted(conf.email_sendinblue_thresholds, reverse=True)
```

Comment:
This looks invalid, as it was recently replaced by #989

Reply:
Oh, yes that shouldn't be there...